pycodestyle.py

```diff
@@ -1554,6 +1561,8 @@ def check_logical(self):
         if self.logical_line:
             self.previous_indent_level = self.indent_level
             self.previous_logical = self.logical_line
+            if not self.indent_level:
+                self.previous_logical_toplevel = self.logical_line
```

Comment:
Perhaps I'm weird, but I think `previous_toplevel_logical_line` is better. (Frankly I never liked `previous_logical` as a name.)